packages/issm/package.py

```diff
@@ -1,126 +1,195 @@
 # Copyright Spack Project Developers. See COPYRIGHT file for details.
 #
 # Copyright 2023 Angus Gibson
-# Justin Kin Jun Hew, 2025
+# Justin Kin Jun Hew, 2025 – Wrappers, Examples, Versioning, AD‑enabled flavour
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
 
 
 class Issm(AutotoolsPackage):
-    """Ice-sheet and Sea-Level System Model"""
+    """Ice‑sheet and Sea‑Level System Model.
+
+    This recipe supports two distinct build flavours:
+
+    * **Classic** (default) – links against PETSc for linear algebra.
+    * **Automatic Differentiation** (+ad) – uses CoDiPack + MediPack and
+      **excludes PETSc** (ISSM’s AD implementation is not PETSc‑compatible).
+    """
 
     homepage = "https://issm.jpl.nasa.gov/"
-    git = "https://github.com/ACCESS-NRI/ISSM.git"
+    git      = "https://github.com/ACCESS-NRI/ISSM.git"
```

Comment:
Please don't make these types of changes. Spack appears to standardise on the `black` linter. Perhaps, run `black` over your file and stick to that formatting.